### PR TITLE
Fixes #10836: Deploy katello.local.rb from a template.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@stbenjam please review

File: manifests/config.pp [hunk 1]
```diff
@@ -13,11 +13,11 @@
   }
 
   file { "${katello_devel::deployment_dir}/foreman/bundler.d/katello.local.rb":
-    ensure => link,
-    target => "${katello_devel::deployment_dir}/katello/doc/katello.local.rb",
-    owner  => $katello_devel::user,
-    group  => $katello_devel::group,
-    mode   => '0644',
+    ensure  => file,
+    content => template('katello/katello.local.rb'),
```
Comment (stbenjam) [inline manifests/config.pp]: Should be `katello_devel`

**Comment by ehelms** [comment]
Good catch... updated

File: manifests/config.pp [hunk 2]
```diff
@@ -13,11 +13,11 @@
   }
 
   file { "${katello_devel::deployment_dir}/foreman/bundler.d/katello.local.rb":
-    ensure => link,
-    target => "${katello_devel::deployment_dir}/katello/doc/katello.local.rb",
-    owner  => $katello_devel::user,
-    group  => $katello_devel::group,
-    mode   => '0644',
+    ensure  => file,
+    content => template('katello_devel/katello.local.rb'),
```
Comment (stbenjam) [inline manifests/config.pp]: Should be katello.local.rb.erb, too :-)

**Comment by stbenjam** [comment]
APT

**Pull request merged**